tests/test_result_storage.py

```diff
@@ -26,83 +24,48 @@ class Request(object):
 class S3StorageTestCase(S3MockedAsyncTestCase):
 
     @gen_test
-    def test_can_store_image(self):
```

Comment:
Why did you delete both tests? They do not apply anymore?

Reply:
They are covered by next 2 tests, which store and get images in one shot.

Reply:
It can be interesting to have specific tests (unitary) to ease out bug resolution. However this may be factorized.